src/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './App.css';
+import Home from './Home'
+
+function App() {
+  return (
+    <div>
+      <Home></Home>
```

Comment:
Not really a big deal but you can make your component tags shorter by using self closing syntax:
`<Home />`

Reply:
Oh that's true, should be the autocomplete, but thank!!